src/hooks/useUserMarketInfo.ts

```diff
@@ -0,0 +1,183 @@
+import { useQuery, useQueries, UseQueryResult } from 'react-query';
+import BigNumber from 'bignumber.js';
+import getMarkets, { GetMarketsOutput } from 'clients/api/queries/getMarkets';
+import getAssetsInAccount from 'clients/api/queries/getAssetsInAccount';
+import getVTokenBalancesAll, {
+  IGetVTokenBalancesAllOutput,
+} from 'clients/api/queries/getVTokenBalancesAll';
+import getHypotheticalAccountLiquidity from 'clients/api/queries/getHypotheticalAccountLiquidity';
+import { TREASURY_ADDRESS } from 'config';
+import FunctionKey from 'constants/functionKey';
+import { useComptroller, useVenusLens } from 'hooks/useContract';
+import { useVaiUser } from 'hooks/useVaiUser';
+import { Asset, Market } from 'types';
+import { indexBy } from 'utilities/common';
+import { VBEP_TOKENS, CONTRACT_TOKEN_ADDRESS, getVbepToken } from 'constants/contracts';
+
+const useUserMarketInfo = ({ account }: { account: string | null | undefined }) => {
+  const comptrollerContract = useComptroller();
+  const venusLensContract = useVenusLens();
+  const { userVaiMinted } = useVaiUser();
+
+  const vtAddresses = Object.values(VBEP_TOKENS)
+    .filter(item => item.address)
+    .map(item => item.address);
+  const { data: markets } = useQuery(FunctionKey.GET_MARKETS, () => getMarkets(), {
+    placeholderData: [],
+  });
+  const { data: assetsInAccount } = useQuery(
+    FunctionKey.GET_ASSETS_IN_ACCOUNT,
+    () => getAssetsInAccount({ comptrollerContract, account }),
+    { placeholderData: [], enabled: !!account },
+  );
+  const { data: vTokenBalancesAccount } = useQuery(
+    [FunctionKey.GET_VTOKEN_BALANCES_ACCOUNT, vtAddresses, account],
+    () => getVTokenBalancesAll({ venusLensContract, vtAddresses, account }),
+    { placeholderData: [], enabled: !!account },
+  );
+  const { data: vTokenBalancesTreasury } = useQuery(
+    [FunctionKey.GET_VTOKEN_BALANCES_ACCOUNT, TREASURY_ADDRESS, account],
+    () => getVTokenBalancesAll({ venusLensContract, vtAddresses, account: TREASURY_ADDRESS }),
+    { placeholderData: [], enabled: !!account },
+  );
```

Comment:
Let's move each of these into separate files too, so that they are reusable.

Reply:
I've done this. I didn't get a chance to finish updating